core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -175,22 +177,89 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
     private fun setCivRankingsTable() {
         val majorCivs = gameInfo.civilizations.filter { it.isMajorCiv() }
-        val civRankingsTable = Table().apply { defaults().pad(5f) }
+        val playerCiv = gameInfo.civilizations.filter { it.isCurrentPlayer() }
+
+        contentsTable.clear()
+
+        if (gameInfo.gameParameters.demographicsEnabled) contentsTable.add(buildDemographicsTable(majorCivs, playerCiv))
+        else contentsTable.add(buildRankingsTable(majorCivs))
+    }
+
+    private fun buildDemographicsTable(majorCivs: List<CivilizationInfo>, playerCiv: List<CivilizationInfo>): Table {
+        val rankLabels = arrayOf("Demographic", "Rank", "Value", "Best", "Average", "Worst")
+        val demographicsTable = Table().apply { defaults().pad(5f) }
+        for (n in rankLabels.indices)   {
+            if (n == 0) {
+                val demoLabel = Table().apply { defaults().pad(5f) }
+                demoLabel.add(rankLabels[n].toLabel()).row()
+                demoLabel.addSeparator().fillX()
+                demographicsTable.add(demoLabel)
+                for (category in RankingType.values()) {
+                    val headers = Table().apply { defaults().pad(5f) }
+                    headers.add(category.name.replace('_', ' ').toLabel()).row()
+                    headers.addSeparator().fillX()
+                    demographicsTable.add(headers)
+                    }
+            } else {
+                demographicsTable.row()
+                demographicsTable.add(rankLabels[n].toLabel())
+                for (category in RankingType.values()) {
+                    when (n) {
+                        1 -> { //finds the current player's rank in each category
+                            for ((rankCount, civ) in majorCivs.sortedByDescending { it.getStatForRanking(category) }.withIndex()) {
```

Comment:
All you need is the current player, so just currentPlayer.getStatForRanking()

Reply:
Done.